desci-server/src/services/manifestRepo.ts

```diff
@@ -187,6 +188,23 @@ export const getNodeManifestUpdater = (node: Node) => {
           { time: Date.now(), message: action.type },
         );
         break;
+      case 'Set Drive Clock':
+        handle.change(
+          (document) => {
+            if (document.driveClock && document.driveClock === action.time) return; // Don't update if already the latest
+            document.driveClock = action.time;
```

Comment:
what happens if the `document.driveClock` is initial undefined for existing documents?

I'd suggest add this before the line
```typescript
if (!document.driveClock) document.driveClock = 0; // register a new field and type in automerge
```